api/openapi_server/openapi/openapi.yaml

```diff
@@ -16,11 +16,13 @@ paths:
   /host:
     $ref: "./paths/host.yaml"
   /auth/signup/host:
-    $ref: "./paths/auth/authSignupHost.yaml"
+    $ref: "./paths/auth/authSignUpHost.yaml"
   /auth/signup/coordinator:
-    $ref: "./paths/auth/authSignupCoordinator.yaml"
+    $ref: "./paths/auth/authSignUpCoordinator.yaml"
   /auth/signin:
     $ref: "./paths/auth/authSignin.yaml"
+  /auth/resend_confirmation_code:
```

Comment:
is this suppose to be snake or camel case

Reply:
Hi @ju1es! Thank you for taking a look. This is a good and important question. 

### Regarding API endpoint (OpenAPI paths):
I checked again. I see there are both ways being used in this file (e.g. `/serviceProviders`, `/auth/initialInvite`, and `/auth/forgot_password`, `/auth/forgot_password/confirm`). 

* When I searched for guidance in the OpenAPI documentation, I didn't see any explicit convention written, but I did see they used camelCase (this is called mixed case in Python PEP8) in their examples. 
* I then checked https://restfulapi.net/resource-naming/ which only says to be consistent in whatever convention is used and provides "hints" to use hyphen (-) to separate words, avoid underscores, and use lowercase letters. 
* Then, I checked the [connexion documentation](https://connexion.readthedocs.io/en/stable/routing.html#endpoint-routing-to-your-python-views) and saw they had an example using snake case (`/hello_world`) (doh!). 
* Finally, I tried searching for preferences from the React side but couldn't find anything.

Do you, @erikguntner, @Joshua-Douglas  know if there is a preference on the front end side regarding the API endpoint naming convention?

### Regarding OpenAPI operationId: 
I tend to use snake case to follow [PEP8 function and variable name convention](https://peps.python.org/pep-0008/#function-and-variable-names) because it's implemented in Python and the `auth_controller.py` is dominantly snake case.

Reply:
@paulespinosa Yeah I was using snake case since that's what I saw in the Flask and connexion docs, but I defer to you all with more Python experience for what you wanna do moving forward. Camel case is the standard in Javascript land and always appreciated when getting data back from a db, but the endpoints don't necessarily need to be written in it. Hyphens look fairly clean tho

Reply:
well, at least we have options. thanks for looking into it so thoroughly @paulespinosa!

+1 hyphens for me as well. i don't have a strong preference except that we adhere to one convention for consistency.